docs/meshstack.authorization.md

```diff
@@ -0,0 +1,84 @@
+
+#### OpenStack
+
+As OpenStack has to scope authorization to a sepcific project, the token from the meshIdB is also scoped to a specific project in the request. The project information can then be read from the **MC_PROJECTS** claim in the token provided by the meshIdB. OpenStack maps this attribute to a local Group associated with the project, that contains the previously mentioned roles.
+
+Project Users get the following OpenStack roles:
+
+- _member_
+- heat_stack_owner
+- creator
+
+The actual access rights associated with these roles are managed by the OpenStack instance and are not part of meshStack.
+
+More details about the User Federation with OpenStack can be found [here](meshstack.openstack.md).
+
+### Authorization via replication
+
+For platforms that don't support the [Authorization via OIDC](#authorization-via-oidc), access rights are replicated during project replication. Cloud platforms provide their own ACL system and meshStack configures it as defined in the meshProject. E.g. this could be an assignment of certain roles for a certain project in the cloud platform.
+
+#### Cloud Foundry
+
+When users are replicated to the UAA of Cloud Foundry they always get the “Org User” and “Space Developer” role. With these roles, they are limited to managing resources in their assigned projects. Access management and creation of new projects is not possible with these roles. The actual access rights associated with these roles are managed by Cloud Foundry and are not part of meshStack.
+Cloud Foundry uses the Identity broker only to authenticate users. Authorization is done as described before via roles set by meshStack.
+
+#### AWS
+
+The Project User Roles are mapped to AWS IAM Roles, which are the authorization roles in AWS. The access rights associated with these IAM roles is not part of meshStack.
+
+## Service User
+
+[Service Users](meshcloud.service-user.md) are technical users, that can be created per platform instance in a project. They are mainly used in CI/CD pipelines. They are local platform users and can therefore only be used to access a specific project in a specific cloud platform. The password of such a generated user is only downloaded once when a service user is created. It is a random generated, 24 characters long password generated with Java's SecureRandom class. meshStack does not store this password. It is the user’s responsibility to safely store it. If the password is somehow compromised, the service user can easily be deleted and replaced by a new service user.
+Service Users are available for Cloud Foundry and OpenStack cloud platforms. Other cloud platforms supported by meshStack like AWS or Kubernetes offer superior “native” alternatives to service users (i.e. IAM Policies & Credentials, ServiceAccounts).
+A Service User can be created and deleted by all users assigned to the project. Information about which user created the Service User and therefore is responsible for the user is available in meshStack.
```

Comment:
```suggestion
A Service User can be created and deleted by all users assigned to the project. Information about the Service User creator is available in meshStack.
```
Otherwise sounds a bit clunky.

Reply:
I split it up into two sentences. I think mentioning, that the creator is responsible for the secure usage of teh service user is a good remark.